en/products/vpn/landing.ftl

```diff
@@ -112,15 +112,13 @@ vpn-landing-faq-compatibility-question-desc-windows-v2 = <a href="{ $url }">{ -b
 
 # Variables:
 #   $url (url) - link to https://www.mozilla.org/products/vpn/desktop/mac/
-vpn-landing-faq-compatibility-question-desc-mac-v3 = <a href="{ $url }">{ -brand-name-mac }</a> (10.14 and up)
+#   $minversion (string) - minimum supported version e.g. 10.15
+vpn-landing-faq-compatibility-question-desc-mac-v4 = <a href="{ $url }">{ -brand-name-mac }</a> ({ $minversion } and up)
```

Comment:
For third party brand names, we stop putting them in placeholders, so just spell it like normal text. 

For the reference tag, `<a href="{ $url }">`, we suggest to use `<a { $url }>` as the extra codes often generates extra errors in localized strings.